components/notebook-dockerfiles/pytorch/requirements.txt

```diff
@@ -1,15 +1,15 @@
 # These packages are installed for both python2 and python3
 
 # Jupyter related packages
-jupyter-console==6.4.4
-jupyterlab==3.4.4
-jupyterlab-git==0.38.0
-notebook==6.4.12
-ipykernel==6.15.1
+jupyter-console==6.6.3
```

Comment:
revert? these should only be updated with notebook images are built

Reply:
reverted